src/messenger-main.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
This seems like "main.tsx" to me. What's different b/w `main.tsx` & `messenger-main.tsx`? 
Assuming all zOS is, is messenger right now 🤷‍♂️

Reply:
Wouldn't `main.tsx` would clash with the existing `Main.tsx` ?